pkg/util/utils_test.go

```diff
@@ -136,3 +139,87 @@ func TestFriendlyDuration(t *testing.T) {
 		}
 	}
 }
+
+func Test_command_executes_successfully_with_absolute_path(t *testing.T) {
+	currentDir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	dummyBinaryPath := filepath.Join(currentDir, "..", "..", "test", "testdata")
+
+	switch os := runtime.GOOS; os {
+	case "darwin":
+		dummyBinaryPath = filepath.Join(dummyBinaryPath, "dummybinary_darwin")
+	case "linux":
+		dummyBinaryPath = filepath.Join(dummyBinaryPath, "dummybinary_linux")
+	case "windows":
+		dummyBinaryPath = filepath.Join(dummyBinaryPath, "dummybinary_windows.exe")
+	default:
+		t.Fatal("Unpexpected OS")
+	}
+
+	testData := []struct {
+		command string
+		exists  bool
+	}{
+		{"blahh", false},
```

Comment:
Let's make this an absolute path to fit the test,

    {filepath.Join(currentDir, "..", "..", "test", "testdata", "blahh"), false},

Other than that, I am ok now.

Reply:
Okay